defi/src/protocols/data.ts

```diff
@@ -12777,18 +12777,17 @@ The eWIT token is a custodial, wrapped version of the Witnet coin managed by the
     address: "arbitrum:0x1622bF67e6e5747b81866fE0b85178a93C7F86e3",
     symbol: "UMAMI",
     url: "https://umami.finance/",
-    description: "The premier market-maker and liquidity network natively built on Arbitrum.",
+    description: "Sustainable, Risk-Hedged Yields on Arbitrum.",
     chain: "Arbitrum",
     logo: `${baseIconsUrl}/umamifinance.png`,
     audits: "0",
     audit_note: null,
     gecko_id: "umami-finance",
     cmcId: "15480",
-    category: "Reserve Currency",
+    category: "Yield",
     chains: ["Arbitrum"],
     module: "umamifinance/index.js",
     twitter: "UmamiFinance",
-    forkedFrom: ["Olympus DAO"],
```

Comment:
isn't it still an ohm fork?

Reply:
Nope, there are no rebases or bonds anymore. It's a fixed supply token with no emissions now.